app/javascript/dashboard/components/ChatList.vue

```diff
@@ -2,7 +2,7 @@
   <div class="conversations-sidebar  medium-4 columns">
     <slot></slot>
     <div class="chat-list__top">
-      <h1 class="page-title">
+      <h1 class="page-title text-truncate">
```

Comment:
I guess it would be good to add a tooltip.

Reply:
![v_tooltip](https://user-images.githubusercontent.com/19498820/100109068-7b60a900-2e91-11eb-826b-2c67da76404d.png)
I tried v-tooltip it doesn't show the whole text  
  
will the title attribute will do?
it is also used in the sidebar label list
![title_tooltip](https://user-images.githubusercontent.com/19498820/100109252-a8ad5700-2e91-11eb-9fa4-41993cd78368.png)

Reply:
`title` should be enough.

Reply:
Cool, I have committed the changes.